#160120906  Heroku hosting with a database

Frediflexta opened the pull request:
### What does this PR do?
> Hosts on heroku with a database
#### Description of Task to be completed?
>edit config file
>edit script to run migrations without resetting the environment to my local host
#### How should this be manually tested?
> Visit `https://freddie-stackoverflowlite.herokuapp.com/api/v1/` and test out the rest of the routes from postman
#### Any background context you want to provide?
> The app i have hosted on heroku at the moment is not persistant with data
#### What are the relevant pivotal tracker stories?
> #160120906
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

Frediflexta committed:
chore(heroku hosting): edit config file

edit config file
add script to run

[#160120906]

Dheavyman commented:
Nice to see you make effort to follow the convention for PR description. However, the hyphen after the story_id is not part of it and should be removed. Your branch name is not according to the convention, why?

Frediflexta commented:
Yikes! oh dear. although if you check all my branches, they all follow the naming convention, this was a silly mistake. i won't give any excuse, so sorry, but going forward, this will not repeat itself

Frediflexta committed:
chore(host on heroku): integrate a database for persisting information

refactor config file
add scripts that transpile to es5 before building
fix tests to pass

[##160120906]

Frediflexta committed:
chore(host on heroku): integrate database with the app hosted on heroku

set environment variable
refactor config file to be readable
refactor a message output

[#160120906]
